### PR TITLE
[C-959] Migrate tipping sagas

### PR DESCRIPTION
**Pull request by sliptype**
### Description

* Migrate tipping sagas to web/common
* Migrate Tipping service to common

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/src/services/audius-backend/audius-backend-instance.ts [hunk 1]
```diff
@@ -49,13 +49,13 @@ export const audiusBackendInstance = audiusBackend({
...
 
-    if (useMetaMask && window.web3) {
+    if (useMetaMask && window.Web3) {
       try {
         return {
           error: false,
           web3Config: await libs.configExternalWeb3(
             registryAddress,
-            window.web3.currentProvider,
+            window.Web3.currentProvider,
```
Comment (dylanjeffers) [inline packages/web/src/services/audius-backend/audius-backend-instance.ts:58]: nice catch

**Review by dylanjeffers — APPROVED** [review]
sweet all makes sense

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sk-migrate-tipping-sagas

File: packages/web/src/services/audius-backend/audius-backend-instance.ts (hunk 1 above)
Comment (sliptype) [reply packages/web/src/services/audius-backend/audius-backend-instance.ts:58]: thanks typescript!

**Pull request merged**